seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/CoordinatorService.java

```diff
@@ -519,24 +524,129 @@ public void printExecutionInfo() {
         int poolSize = threadPoolExecutor.getPoolSize();
         long completedTaskCount = threadPoolExecutor.getCompletedTaskCount();
         long taskCount = threadPoolExecutor.getTaskCount();
+
+        AtomicLong createdJobCount = new AtomicLong();
```

Comment:
I think maybe we should change the time to print the info. If only print when job done, streaming job will never print this info.

Reply:
I understand that this is a server-side tasks regularly, have nothing to do with specific jobs.

Reply:
> I think maybe we should change the time to print the info. If only print when job done, streaming job will never print this info.

Done.